### PR TITLE
{devel}[foss/2022a] Spark v3.3.1 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by surak**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/Spark/Spark-3.3.1-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Author: Denis Krišťák (INUITS)
+
+easyblock = 'Tarball'
+
+name = 'Spark'
+version = '3.3.1'
+
+
+
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/s/Spark/Spark-3.3.1-foss-2022a.eb:9]: To fix `E303 too many blank lines`:
```suggestion

```

File: easybuild/easyconfigs/s/Spark/Spark-3.3.1-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    'https://downloads.apache.org/%(namelower)s/%(namelower)s-%(version)s/'
+]
+sources = ['%(namelower)s-%(version)s-bin-hadoop3.tgz']
+checksums = ['91df3e3f73682d32261a8b245af6b0553dc5cf4af208ef82219283d23ee01ece']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Java', '11', '', SYSTEM),
+    ('Arrow', '8.0.0'),
+]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/s/Spark/Spark-3.3.1-foss-2022a.eb:23]: surely this should include `Hadoop`? As per the documentation:
> Get Spark from the [downloads page](https://spark.apache.org/downloads.html) of the project website. This documentation is for Spark version 3.3.1. Spark uses Hadoop’s client libraries for HDFS and YARN. Downloads are pre-packaged for a handful of popular Hadoop versions. Users can also download a “Hadoop free” binary and run Spark with any Hadoop version [by augmenting Spark’s classpath](https://spark.apache.org/docs/3.3.1/hadoop-provided.html).

Comment (jfgrimm) [reply easybuild/easyconfigs/s/Spark/Spark-3.3.1-foss-2022a.eb:23]: d'oh

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
himem04.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/68371b48e44145d11008c3fbae7cc606 for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/4f456767692f8606b08d4f3b0d4b910c for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Pull request merged**